src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -4,6 +4,10 @@ Release History
 ===============
 * functionapp: az functionapp devops-build, new command created
 
+0.2.14
+++++++
+* webapp, functionapp: az webapp/functionapp deployment list-publishing-credentials, get the Kudu (scm) url and its credentials
```

Comment:
Can you move this change log to top? and then I will merge

Reply:
changed.